test/__snapshots__/index.js.snap

```diff
@@ -1095,21 +1095,6 @@ Array [
     "status": 200,
     "to": "/.netlify/functions/___netlify-handler",
   },
-  Object {
-    "from": "/api/enterPreview",
-    "status": 200,
-    "to": "/.netlify/functions/_api_enterPreview-handler",
-  },
-  Object {
-    "from": "/api/exitPreview",
-    "status": 200,
-    "to": "/.netlify/functions/_api_exitPreview-handler",
-  },
-  Object {
-    "from": "/api/hello",
-    "status": 200,
-    "to": "/.netlify/functions/_api_hello-handler",
-  },
   Object {
```

Comment:
Only generate custom rewrites for advanced API routes (background and scheduled functions).

Reply:
Where are we mapping the other API routes to the SSR handler?

Reply:
From what I understood of the previous implementation, the `/api/*` has a handler that catches all other requests. I'll take a peek again when I'm start my day.